docker/dockerfiles/backend.Dockerfile

```diff
@@ -37,11 +37,23 @@ RUN set -e; \
     # source command may not be availble in sh
     . .venv/bin/activate; \
     \
+    # Install opentelemetry for instrumentation.
+    pip install opentelemetry-distro opentelemetry-exporter-otlp; \
+    \
+    opentelemetry-bootstrap -a install; \
+    \
+    # Applicaiton dependencies.
     pdm sync --prod --no-editable; \
     \
     # REF: https://docs.gunicorn.org/en/stable/deploy.html#using-virtualenv
     pip install --no-cache-dir gunicorn;
 
+# Disable all telemetry by default
+ENV OTEL_TRACES_EXPORTER none
+ENV OTEL_METRICS_EXPORTER none
+ENV OTEL_LOGS_EXPORTER none
+ENV OTEL_SERVICE_NAME unstract_backend
+
```

Comment:
@ritwik-g Can we move up these envs also to the top along with other existing env definitions?

Reply:
@hari-kuriakose done